.github/workflows/apk-builder.yml

```diff
@@ -0,0 +1,89 @@
+name: APK build
+on:
+  workflow_dispatch:
+
+permissions:
+    contents: write
+      
+jobs:
+    build:
+      name: Build Release APK
+      runs-on: ubuntu-latest
+  
+      steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+  
+      - name: Set up JDK
+        uses: actions/setup-java@v4
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+
+      # Load the google-services.json file from the secrets
+      - name: Load Google Service file
+        env:
+          DATA: ${{ secrets.GOOGLE_SERVICES_JSON }}
```

Comment:
typo: the current secret for the google services json file is called GOOGLE_SERVICES and not GOOGLE_SERVICES_JSON